pykern/util.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""utils for pykern
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern import pkconfig
+from pykern import pkinspect
+from pykern import pkio
+from pykern.pkdebug import pkdc, pkdexc, pkdlog, pkdp
+import os.path
+import sys
+
+
+_DEFAULT_ROOT = "run"
+
+
+def cfg_db_dir(db_root_env_name):
+    """Generate pkconfig parser function with errors for db's env var name
+
+    Args:
+        db_root_env_name (str): env var name for run dir root
+
+    Returns:
+        function: parser function for pkconfig.init
+    """
+    _NOT_ABS_ERROR = "{}: " + f"{db_root_env_name} must be absolute"
```

Comment:
Good solution, but db_root_env_name shouldn't need to be passed. Rather https://github.com/radiasoft/pykern/issues/74 should be fixed to always print the key on errors. This would simplify this code, since the error messages wouldn't need to print the key.